### DISCUSSION TITLE
Unable to insert a prompt?

### DISCUSSION THREAD

**Discussion by madscientista** [betalgo/openai#274] [Q&A]
**Describe the bug**
I am getting this error while trying to build it
```
error CS0117: 'ChatCompletionCreateRequest' does not contain a definition for 'Prompt'
```

```
var completionResult = await openAiService.ChatCompletion.CreateCompletion(
            new ChatCompletionCreateRequest
            {
                Prompt = "Something here",
                Messages = messages,
                Model = Models.ChatGpt3_5Turbo,
                MaxTokens = 66,
                User = userId
            },
            cancellationToken: cancellationToken
        );

```

**Expected behavior**
Use the prompt

**Desktop (please complete the following information):**
- Linux Ubuntu 22.04 Server

**Additional context**
How would I pass in the prompt using that function?

**Comment by kayhantolga** [comment]
Please check the wiki page https://github.com/betalgo/openai/wiki/Chat-GPT.

The error message is accurate. You need to use "message" instead of "prompt". If this is a sample that I shared before in the repository, please let me know so I can update it.

**Reply by madscientista** [reply]
> Please check the wiki page https://github.com/betalgo/openai/wiki/Chat-GPT.
> 
> The error message is accurate. You need to use "message" instead of "prompt". If this is a sample that I shared before in the repository, please let me know so I can update it.

Thank you! But unfortunately it still says `error CS0117: 'ChatCompletionCreateRequest' does not contain a definition for 'message'`

I was also wondering what models I can use? Can't seem to use anything other than GPT 3.5?

**Reply by kayhantolga** [reply]
I highly suggest checking the Wikipedia page I shared. The error message is accurate: instead of `message`, please use `Messages` . Additionally, please refer to the OpenAI documentation for available models at https://platform.openai.com/docs/models/overview.
